code/datums/abnormality/_ego_datum/zayin.dm

```diff
@@ -106,3 +106,29 @@
 	item_path = /obj/item/clothing/suit/armor/ego_gear/zayin/letter_opener
 	cost = 12
 
+// Wishing Well - Bucket
```

Comment:
Bucket already exists as an E.G.O. under the "tools" section. Remove all new instances of the E.G.O. as they are just a duplicate with nonfunctional icons.